### PR TITLE
v0.0.1-beta

### PR DESCRIPTION
**Pull request by mdzhang**
- Add ability to exclude select tasks
- MIT license
- CLI tool (#5)
- resolves a docker native compatibility issue (#6)

### CODE REVIEW

File: lib/docker_rake.rb [hunk 1]
```diff
@@ -4,4 +4,23 @@ module DockerRake
...
+
+  def self.tasks_to_load_paths
+    path = File.join(File.expand_path(__dir__), 'docker_rake', 'tasks')
+    excluded_tasks = Helper.load_docker_rake_file&.dig('tasks', 'exclude')
+    all_tasks = Dir.glob("#{path}/**/*.rake")
+
+    if excluded_tasks.nil?
+      all_tasks
+    else
+      all_tasks - excluded_tasks.map do |full_task_name|
```
Comment (alexQueue) [inline lib/docker_rake.rb]: Maybe split this into multiple lines. It's a little confusing right now

File: spec/docker_rake_spec.rb [hunk 2]
```diff
@@ -7,4 +7,63 @@
...
+
+  describe '.tasks_to_load_paths' do
+    subject { DockerRake.tasks_to_load_paths }
+
+    before do
+      allow(DockerRake::Helper).to receive(:load_docker_rake_file).and_return(docker_rake_config)
+    end
+
+    let(:namespace) { 'docker' }
+    let(:tasks) { ['docker:psql', 'docker:view'] }
```
Comment (alexQueue) [inline spec/docker_rake_spec.rb]: These are just the excluded tasks, right?

Comment (mdzhang) [inline spec/docker_rake_spec.rb]: correct

Comment (alexQueue) [inline spec/docker_rake_spec.rb]: NB I'd make the variable name more descriptive then

**Comment by mdzhang** [comment]
@alexQueue Feedback addressed.

**Pull request merged**